diff --git a/dist/index.js b/dist/index.js
index 711283b40632ee5b8dcdda3c497954a3a230513f..db23d82c954e435a41d5e262208dcc40f9ec9ed2 100644
--- a/dist/index.js
+++ b/dist/index.js
@@ -427,9 +427,6 @@ function connect(state, send, normalize) {
         send("BLUR");
       },
       onChange(event) {
-        const evt = getNativeEvent(event);
-        if (evt.isComposing)
-          return;
         send({ type: "CHANGE", value: event.currentTarget.value });
       },
       onKeyDown(event) {
diff --git a/dist/index.mjs b/dist/index.mjs
index 8fed36613f84775a4779d4a7ee59d1ef555d9703..d64712ffb1774b87a842d5e354a09a89ab610ab0 100644
--- a/dist/index.mjs
+++ b/dist/index.mjs
@@ -399,9 +399,6 @@ function connect(state, send, normalize) {
         send("BLUR");
       },
       onChange(event) {
-        const evt = getNativeEvent(event);
-        if (evt.isComposing)
-          return;
         send({ type: "CHANGE", value: event.currentTarget.value });
       },
       onKeyDown(event) {